### PR TITLE
db properties

### PR DESCRIPTION
**Pull request by gorshkovayulia**
Move db properties from controllers-context to application properties file

### CODE REVIEW

**Comment by gorshkovayulia** [comment]
@ctapobep please review

**Comment by gorshkovayulia** [comment]
@ctapobep I see some warnings that IDEA _cannot resolve property key_, but everything works fine and maybe the problem is IDEA

File: src/main/resources/controllers-context.xml [hunk 1]
```diff
@@ -18,12 +19,14 @@
...
-        <property name = "url" value = "jdbc:h2:mem://localhost:8081/;DB_CLOSE_DELAY=-1"/>
-        <property name = "username" value = "admin"/>
-        <property name = "password" value = "admin"/>
+        <property name = "driverClassName" value = "${db.driverClassName}"/>
+        <property name = "url" value = "${db.url}"/>
+        <property name = "username" value = "${db.username}"/>
+        <property name = "password" value = "${db.password}"/>
     </bean>
 
+    <context:property-placeholder location="file:C:\Users\Iuliia_Gorshkova\Desktop\Dog\src\main\resources\application.properties"/>
```
Comment (ctapobep) [inline src/main/resources/controllers-context.xml]: How will I run this? :)

Comment (gorshkovayulia) [reply src/main/resources/controllers-context.xml]: oops, fixed

File: src/main/resources/controllers-context.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
         <property name = "password" value = "${db.password}"/>
     </bean>
 
-    <context:property-placeholder location="file:C:\Users\Iuliia_Gorshkova\Desktop\Dog\src\main\resources\application.properties"/>
+    <context:property-placeholder location="file:application.properties"/>
```
Comment (ctapobep) [inline src/main/resources/controllers-context.xml]: Um.. Did you check it? "file:" means it's going to look for the file on file system in your working directory. You need to look for it in the classpath.

**Comment by ctapobep** [comment]
+2

**Pull request merged**